src/minisatip.c

```diff
@@ -1669,23 +1670,18 @@ int close_http(sockets *s) {
 #define DEFAULT_LOG LOG_SSDP
 
 int ssdp_notify(sockets *s, int alive) {
-    char buf[500], mac[15] = "00000000000000";
+    char buf[500];
     char nt[3][60];
     int ptr = 0;
     char *op = alive ? "Discovery" : "ByeBye";
 
-    char uuid1[] = "11223344-9999-0000-b7ae";
     int i;
     s->wtime = getTick();
-    if (uuidi == 0) {
-        uuidi = 1;
-        get_mac_address(mac);
-        sprintf(uuid, "%s-%s", uuid1, mac);
-        // use IPv4 only as disc_host is multicast IPv4
-        fill_sockaddr(&ssdp_sa, opts.disc_host, 1900, 1);
-    }
+
+    // use IPv4 only as disc_host is multicast IPv4
+    fill_sockaddr(&ssdp_sa, opts.disc_host, 1900, 1);
```

Comment:
I have a feeling that this is no longer correct (this should not be called all the time)

Reply:
The multicast address needs to be set somewhere in the ssdp_sa. Maybe is not the right place but it should be done, or am I missing something?

Reply:
The fact that the variable `ssdp_sa` was global made me think there were some other uses for it, but it's apparently only used in this function so it should be fine to call `fill_sockaddr` many times.